### DISCUSSION TITLE
Include a modules connection graph

### DISCUSSION THREAD

**Discussion by kodiweera** [nipreps/fmriprep#2517] [Ideas]
Hi,
I think it's nice to include a modules connection graph of the project which will help new devs to quicikly grasp the flow of the project. I created such a graph for fmriprep using pydeps. 

![fmriprep](https://user-images.githubusercontent.com/6729492/130712697-d7479f24-b2ff-465e-8da7-28120abc271b.png)

This is a figure and will readily visible if included in the main directory wihout much work. However, whenever a new module is added to the project, a new graph shoud be genrated.

Chandana Kodiweera

**Comment by oesteban** [comment]
It could be interesting to add this graph to the developer's section of the documentation, and have it regenerate with every documentation build.
